### PR TITLE
Added blog post about the book.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: bwplotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
Preview: https://6378c15f82dd304a02a34fb7--bwplotka.netlify.app/

File: web/content/posts/efficient-go-release.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+* Amazon ([US](https://www.amazon.com/Efficient-Go-Data-Driven-Performance-Optimization/dp/1098105710), [UK](https://www.amazon.co.uk/Efficient-Go-Data-Driven-Performance-Optimization/dp/1098105710?crid=XH6ZOQIV3IYH&keywords=Efficient+Go&qid=1662479068&s=books&sprefix=efficient+go%2Cstripbooks%2C152&sr=1-2&linkCode=ll1&tag=bwplotka-21&linkId=3c2d7389f9790829cf6bc46d6165f9b3&ref_=as_li_ss_tl), [DE](https://www.amazon.de/Efficient-Go-Data-Driven-Performance-Optimization/dp/1098105710/))
+* [Barnes & Noble](https://www.barnesandnoble.com/w/efficient-go-bartlomiej-plotka/1141565108?ean=9781098105716)
+
+E-books are available. If you order a printed copy, be mindful that Amazon is scaring customers with quite a long print & delivery time (~one month during Christmas). Those things are outside of my control, but I heard they tend to overestimate the timing for books, so let's be hopeful! 🤞
+
+### Feedback
+
+I am already getting good feedback from people who already managed to read some of it (and it was about the 9th chapter, so either they skipped a lot or read very fast! 🙈). There is nothing better than good feedback--it's essentially one reason why I create this book. (:
```
Comment (ivan-valkov) [inline web/content/posts/efficient-go-release.md:58]: too many `alreadys` :D
```suggestion
I am already getting good feedback from people who managed to read some of the book (and it was about the 9th chapter, so either they skipped a lot or read very fast! 🙈). There is nothing better than good feedback--it's essentially one reason why I create this book. (:
```

File: web/content/posts/efficient-go-release.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+> [Ivan on LinkedIn](https://www.linkedin.com/feed/update/urn:li:activity:6997949174461652992?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A6997949174461652992%2C6997996313740984321%29
+): It will be awesome to get some snippets from your book here. I’m sure this way more people will get to know about it as well 😛
+
+Why not! It's hard to tell which chapter is my favourite, but I am especially proud of Chapter 9, "Bottleneck Analysis, " which covers Profiling in detail. Those techniques allow us to tell exactly the **main** reason for slowdowns or high consumption usage, e.g. memory in your application. Most likely, that part is the most impactful thing you can change to improve efficiency! Here are two pages (out of 50 in this chapter) discussing heap profiles easily gatherable in Go. There are many less-known details about them--knowing them makes you so much more effective.
+
+![](/images/blog/efficient-go/heap.png)
+
+At the end of this chapter you will learn how to setup continous profiling setup for **free** in minutes, using amazing open source software like [Parca](https://github.com/parca-dev/parca). And it's not just a "toy"--we use it in production for ~2 years. 😉
```
Comment (ivan-valkov) [inline web/content/posts/efficient-go-release.md:44]:
```suggestion
At the end of this chapter you will learn how to setup continous profiling setup for **free** in minutes, using amazing open source software like [Parca](https://github.com/parca-dev/parca). And it's not just a "toy"--we have been using it in production for ~2 years. 😉 
```

**Review by ivan-valkov — APPROVED** [review]
awesome

**Review by bwplotka — COMMENTED** [review]
Thanks!

**Pull request merged**